### ISSUE TITLE
Bn Problems

### ISSUE THREAD

**Issue by fhwzl** [pytorch/xla#2570]
hi, same as #900, I have the problem too with batchnorm enable. But I need use the torch-xla:0.1, anyone know the reason? thx

**Comment by zcain117** [comment]
Are you saying that the problem is fixed if you use 0.1?
Or that there is some reason you have to use 0.1 and you wanted batchnorm to work on 0.1?

**Comment by fhwzl** [comment]
yes, I used the torch-xla:0.1, is it fixed?

**Comment by zcain117** [comment]
I think there is a good chance that it's fixed. I would recommend trying `pytorch-nightly` TPU with `nightly` version of pytorch/xla and see if it works

**Comment by fhwzl** [comment]
Ok, I know, It's my problem, thx.
